Linode Object Storage (au-mel-1)(e2).cyberduckprofile

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2021 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>linode-au-mel-3</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Linode Object Storage (Melbourne, AUS) (au-mel-1.linodeobjects.com) (Type=E2)</string>
+        <key>Default Hostname</key>
+        <string>au-mel-1.linodeobjects.com</string>
+        <key>Hostname Configurable</key>
+        <false/>
+        <key>Port Configurable</key>
+        <false/>
+        <key>Username Placeholder</key>
+        <string>Access Key</string>
+        <key>Password Placeholder</key>
+        <string>Secret Key</string>
+        <key>Authorization</key>
+        <string>AWS4HMACSHA256</string>
+        <key>Region</key>
+        <string>au-mel-3</string>
+        <key>Regions</key>
+        <array>
+            <string>au-mel-3</string>
+        </array>
```

Comment:
Not `au-mel-1`?

Reply:
Resolved in https://github.com/iterate-ch/profiles/pull/127/commits/d7bacfe9931e302bd0856c1a10a6943eabc640a3

Reply:
Thank-you for calling this out, I missed it